shanoir-uploader/src/main/java/org/shanoir/uploader/model/dto/StudyCardOnStudyResultDTO.java

```diff
@@ -0,0 +1,65 @@
+package org.shanoir.uploader.model.dto;
+
+public class StudyCardOnStudyResultDTO {
```

Comment:
I think it's not a dto. I know that StudyCardOnStudyResult in the datasets ms is in a package named "dto" but I think that package is not well named either.

Reply:
hm, for me it is a DTO that only purpose is to be transferred to the GUI to be displayed?